demisto_sdk/commands/common/errors.py

```diff
@@ -949,6 +950,11 @@ def pack_metadata_isnt_json(pack_meta_file):
     def pack_metadata_missing_url_and_email():
         return 'Contributed packs must include email or url.'
 
+    @staticmethod
+    @error_code_decorator
+    def pack_metadata_invalid_support_type(pack_meta_file):
+        return 'Support field should be one of the following: `xsoar`, `partner` or `community`.'
```

Comment:
what are the backticks for? what does it look like in the cli?

Reply:
it looks like this:
![image](https://user-images.githubusercontent.com/45915502/101342300-075bd300-388b-11eb-98f6-ae1fcb1fdb62.png)

Remove?

Reply:
yea